RestApi/Sepes.RestApi/appsettings.Development.json

```diff
@@ -1,9 +1,22 @@
 {
+  "ApplicationSettings": {
+    "BaseUrl": "localhost",
+    "ApplicationName": "Sepes Restapi",
+    "Version": "0.0.4",
+    "Environment": "Development"
+  },
   "Logging": {
     "LogLevel": {
       "Default": "Debug",
       "System": "Information",
       "Microsoft": "Information"
     }
+  },
+  "Jwt": {
+    "Key": "secretkey2asfafds56865",
+    "Issuer": "example.com"
+  },
+  "AzureLogToken": {
+    "ServiceApiKey": "AZURETokenHere"
```

Comment:
You should not need to have any config values here that are the same as in appsettings.json